ethereum-explorer/substreams.yaml

```diff
@@ -0,0 +1,43 @@
+specVersion: v0.1.0
+package:
+  name: "substreams_template"
+  version: v0.1.0
+
+protobuf:
+  files:
+    - transaction.proto
+    - event.proto
+    - block_meta.proto
+  importPaths:
+    - ./proto
+
+binaries:
+  default:
+    type: wasm/rust-v1
+    file: ./target/wasm32-unknown-unknown/release/substreams.wasm
+
+modules:
```

Comment:
We should document each module, we should also give "filter" spec for those accepting more than 1 param.